### PR TITLE
Fixes #33635 - update Audits & Models pages buttons to PF4 

### PR DESCRIPTION
**Pull request by Ron-Lavi**

### CODE REVIEW

File: webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js [hunk 1]
```diff
@@ -26,12 +26,13 @@ const AuditsPage = ({
...
+        component="a"
+        className="btn-docs"
         href={getManualURL('4.1.4Auditing')}
-        target="_blank"
         rel="external noreferrer noopener"
-        className="btn-docs"
+        target="_blank"
+        variant="secondary"
       >
-        <Icon type="pf" name="help" />
```
Comment (sharvit) [inline webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js:34]: Is it better without an icon?

**Review by sharvit — CHANGES_REQUESTED** [review]
Thanks @laviro,

I have noticed 2 issues:
1. The bookmark button is 1 pixel off in google-chrome and is ok in safari.
2. The `/` label In the search bar doesn't have enough spacing. (noticed both in chrome and safari)

Screen Shot 2021-10-05 at 11 41 21

File: webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js (hunk 1 above)
Comment (Ron-Lavi) [reply webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js:34]: @sharvit thanks for the review!
The search bar pf4 applied styles is being discussed in #8743 

There was a request to divide the other buttons adjustments to another PR so only the second commit here is relevant :)

**Comment by tbrisker** [comment]
@laviro why should this be blocked by (and include) the other PR? we can update the buttons separately from the search field.
@sharvit the feedback for the search component should go to GH-8743

**Comment by Ron-Lavi** [comment]
> @laviro why should this be blocked by (and include) the other PR? we can update the buttons separately from the search field.

Well design-wise I was thinking it will look odd with an old search bar and new buttons

But since they both are about to go in soon, I have no issue with what order to get them in.

**Comment by Ron-Lavi** [comment]
@tbrisker I made the PR completely separated from the searchbar PR,
although together they look much better:

Audits:

gnome-shell-screenshot-P61HB1

Models:

gnome-shell-screenshot-4K8GB1

**Comment by Ron-Lavi** [comment]
In addition, I am thinking about creating a toolbar component that will contain the search with a button and kebab with margin best practices between all of the components.

File: webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js [hunk 2]
```diff
@@ -26,13 +26,13 @@ const ModelsPage = ({
 
   const createBtn = (
     <Link to="/models/new">
-      <Button bsStyle="primary">{__('Create Model')}</Button>
```
Comment (tbrisker) [inline webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js:29]: The integration test needs to be updated to match the switch to sentence case.

**Comment by Ron-Lavi** [comment]
Updated screenshots
gnome-shell-screenshot-WYAOB1


gnome-shell-screenshot-SUEVB1

**Review by tbrisker — APPROVED** [review]
Thanks @laviro !

**Comment by tbrisker** [comment]
me-- for not noticing the failing test, opened https://github.com/theforeman/foreman/pull/8889 to fix that.

**Pull request merged**